.gitattributes

```diff
@@ -1 +1,3 @@
 * text=auto
+secrets.yaml filter=git-crypt diff=git-crypt
```

Comment:
Should we add `.yml` as well just to be sure?

Reply:
Yea, that would be a good idea to be safe

Reply:
https://github.com/python-discord/kubernetes/pull/156/commits/8ec8aac9b25c67dc221b716d0c3750d0a5a30e20